src/cpp/rtps/reader/StatefulReader.cpp

```diff
@@ -659,10 +659,10 @@ bool StatefulReader::updateTimes(const ReaderTimes& ti)
     return true;
 }
 
-bool StatefulReader::isInCleanState() const
+bool StatefulReader::isInCleanState()
```

Comment:
If `const` was removed because the mutex, make it mutable instead of removing the `const`.

Reply:
We have to discuss this as group.